src/patternfly/components/LabelGroup/label-group.scss

```diff
@@ -0,0 +1,105 @@
+.pf-c-label-group {
+  // label group list
+  --pf-c-label-group__list--MarginBottom: calc(var(--pf-global--spacer--xs) * -1);
+  --pf-c-label-group__list--MarginRight: calc(var(--pf-global--spacer--xs) * -1);
+
+  // category
+  --pf-c-label-group--m-category--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label-group--m-category--PaddingRight: var(--pf-global--spacer--xs);
+  --pf-c-label-group--m-category--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label-group--m-category--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-label-group--m-category--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-label-group--m-category--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-label-group--m-category--BorderColor: var(--pf-global--BorderColor--300);
+  --pf-c-label-group--m-category--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  // Label in group
+  --pf-c-label-group__label--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-label-group__label--MarginBottom: 0;
+  --pf-c-label-group--m-vertical__label--MarginBottom: var(--pf-global--spacer--sm);
+  --pf-c-label-group__label--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-label-group__label--MaxWidth: 18ch;
```

Comment:
in a future issue maybe we could make this a global var since we seem to be using the max-width for labels/text in groups a lot

Reply:
Good idea. @mceledonia @mcarrano wdyt about that? Seems like a nice theming option.